pkg/webhook/configuration.go

```diff
@@ -447,7 +448,7 @@ func preUpgrade(openebsNamespace string) error {
 	}
 
 	for _, service := range svcList.Items {
-		if len(service.Labels["openebs.io/version"]) == 0 {
+		if service.Labels["openebs.io/version"] != version.Current() {
```

Comment:
Why does service need to be deleted for every upgrade?

Reply:
No, we shuold not